frontend/src/layout/navigation/Breadcrumbs/breadcrumbsLogic.tsx

```diff
@@ -58,6 +58,8 @@ export const breadcrumbsLogic = kea<breadcrumbsLogicType>({
                 },
             ],
             (crumbs): Breadcrumb[] => crumbs,
+            null, // PropTypes
+            objectsEqual,
```

Comment:
Didn't know these are a thing

Reply:
`PropTypes` had been there since 2016, but I just added the last `isEqual` parameter to fix this bug. Someone had also requested it a month ago in kea's issues...